rust/src/dcerpc/dcerpc.rs

```diff
@@ -953,19 +953,11 @@ impl DCERPCState {
 
         let buffer = match direction {
             Direction::ToServer => {
-                if self.buffer_ts.len() + input_len > 1024 * 1024 {
```

Comment:
> A limit of 1MB can be easily reached in some of such scenarios

Sure, but was not the purpose of this check to prevent too much CPU consumption in later code ?

Reply:
Oh maybe, it was because we did not want indefinite buffering, but now the TCP code handles it...

Reply:
I thought TCP code could still lead to indefinite buffering and that's where your work about putting bounds on everything helps.. Is this incorrect?
Where does the TCP code handle it?

Reply:
Not sure, but there is stream depth, tcp window and other limiting factors there...